src/main/java/seedu/address/ui/PersonDetailedViewContentManager.java

```diff
@@ -0,0 +1,147 @@
+package seedu.address.ui;
+
+import seedu.address.model.person.Person;
+
+/**
+ * Manages content generation for a {@code Person}'s detailed view,
+ * including strings for fields, templates, and button labels.
+ */
+public class PersonDetailedViewContentManager {
+    private final Person person;
+
+    /**
+     * Constructs a {@code PersonDetailedViewContentManager} with the specified {@code Person}.
+     * @param person The person whose details are displayed.
+     */
+    public PersonDetailedViewContentManager(Person person) {
+        this.person = person;
+    }
+
+    /**
+     * Gets the full name of the person.
+     * @return A formatted string representing the person's full name.
+     */
+    public String getName() {
```

Comment:
Are these methods being used? If not why dont you just use the methods from person class

Reply:
Regarding these methods, I considered anything with strings to be content and I wanted to separate it from PersonDetailedView such that PersonDetailedView would be solely responsible for the generation of the View, while this file would provide all the content.